specs/pvs/corla.pvs

```diff
@@ -1133,6 +1216,13 @@ BEGIN
   random_list_of_ballots:
```

Comment:
I'm modeling everything the C.R.S. specifies.  What we implement is a subset of that functionality.  I will add a @todo in the preface to explain how this works and how the reader can understand exactly what is built vs. what is just designed.